packages/core/src/styles/components/_tooltip.scss

```diff
@@ -79,6 +79,13 @@
 
 				&:not(:last-child){
 					margin-bottom: 2px;
+					@if $carbon--theme == $carbon--theme--g90 {
```

Comment:
can we please add a blank line between `margin-bottom: 2px;` and the if-statement?